ras/servicelog.py

```diff
@@ -115,6 +115,7 @@ def test_servicelog_notify(self):
         ids = self.run_cmd_out("servicelog_notify --list --command=%s | "
                                "grep \"Servicelog ID:\" | cut -d\":\" -f2 | sed 's/^ *//'" %
                                notify_script)
+        ids = ids.split()
```

Comment:
it shoould be `strip()`

Reply:
The type of 'ids' is string, converting to list using 'split'. With this there is no need to use 'strip'.

Reply:
ah ok sure , just last one help can you please update test result with latest changes

Reply:
The result which i pasted above are updated one only praveen.Yesterday evening itself i updated that